content/fr/blog/2021-04-27-cant-read-wont-buy.md

```diff
@@ -1,92 +1,93 @@
 
-> We are going to start seeing some awareness about moving away from the words. Moving away from just putting the source into the target language and just pure translation to start having more questions about how the text might be perceived in China, in Russia, or in Korea.
+Miguel Sepulvida, Directeur de la Mondialisation chez King Games, estime que la traduction n'est qu'un début. Dans un [entretien avec Inlingo](https://blog.inlingogames.com/en/2019/10/29/the-game-has-to-be-super-amazing-for-you-to-forget-about-the-language-barrier-an-interview-with-global-localization-manager-at-king-miguel-sepulveda/), il a déclaré :
 
-King is ahead of the curve &mdash; during the live presentation, an audience member asked Sepulveda whether King had noticed any relationship between investment in globalization and the company's bottom line. The answer was a resounding "Yes!" Clearly for King, creative localization and LQA have been good for business.
+> Nous allons commencer à prendre conscience de la nécessité de s'éloigner des mots. Nous allons commencer à nous interroger sur la façon dont le texte peut être perçu en Chine, en Russie ou en Corée, plutôt que de simplement transposer la source dans la langue cible et de se contenter d'une traduction pure.
 
-## The Future of Localization
+King est en avance sur son temps &mdash; pendant la présentation en direct, un membre de l'auditoire a demandé à Sepulveda si King avait remarqué un lien entre l'investissement dans la mondialisation et les résultats de l'entreprise. La réponse a été un "Oui !" retentissant. Il est clair que pour King, la localisation créative et l'AQL ont été bénéfiques pour les affaires.
 
-What does all mean for app developers who are hoping to release software to geographically distributed audiences? It seems that there are several key challenges that such organizations will have to tackle; from the [technical challenges of distributed systems](https://rotational.io/blog/a-distributed-systems-maze/) to the recognition that English-only may not be a sufficient sales medium &mdash; and perhaps even more importantly, to learning how rich historic contexts and cultural experiences can inform the buying preferences of potential customers around the globe. Afterall, app developers know that the key to growth and impact is trust and engagement with end users; localization is increasingly important to those objectives. King Games proves localization *is* the future. 
+## L'Avenir de la Localisation
```

Comment:
Interesting, it looks like the paras are slightly off; I wonder what would happen if we did a diff of the English and French versions? E.g. is the spacing enough for us to use this as lightweight training data for an NMT model?